packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -111,10 +111,8 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
         });
         this.contextKeyService.onDidChange(e => {
             for (const [, view] of this.views.values()) {
-                if (view.when === undefined) {
-                    continue;
-                }
-                if (e.affects(new Set([view.when]))) {
```

Comment:
Should not we rather fix that an event for a preference change is fired?

Reply:
done